### PR TITLE
removed references to caasp

### PR DESCRIPTION
**Pull request by jcayouette**
# Description

Removed CaaSP support EOL

# Target branches

Which documentation version does this PR apply to?

- [x] Master (Default)
- [ ] Manager-4.2
- [ ] Manager-4.1
- [ ] Manager-4.0

# Links

https://github.com/SUSE/spacewalk/issues/16392

### CODE REVIEW

**Comment by juliogonzalez** [comment]
@jcayouette I am am removing myself as reviewer. Talk to the developers who did the removal.

File: modules/client-configuration/pages/virtualization.adoc [hunk 1]
```diff
@@ -78,294 +77,6 @@ You can add virtual guests on a pacemaker cluster within the {productname} {webu
 
 Virtual Guests managed by a cluster can be live migrated.
 
-////
```
Comment (rjmateus) [inline modules/client-configuration/pages/virtualization.adoc:81]: I'm not sure if this section should be removed. Is this one linked to caasp or virtualization in general?

**Review by rjmateus — COMMENTED** [review]
Just one question

**Comment by rjmateus** [comment]
@jcayouette can you also add @mbologna as a reviewer, since he was one of the developers of the feature?

File: modules/client-configuration/pages/virtualization.adoc (hunk 1 above)
Comment (jcayouette) [reply modules/client-configuration/pages/virtualization.adoc:81]: I thought it was virt in general but it is referring to pacemaker which is a CaaSP reference right?

**Comment by jcayouette** [comment]
Sure, thanks @rjmateus

**Review by keichwa — APPROVED** [review]

**Comment by juliogonzalez** [comment]
@jcayouette @rjmateus is this ready to be merged?

**Comment by rjmateus** [comment]
> @jcayouette @rjmateus is this ready to be merged?

I think we are waiting on feedback from @mbologna about one section which I'm not sure if it should be deleted.

**Review by mbologna — CHANGES_REQUESTED** [review]

**Comment by juliogonzalez** [comment]
Ping @jcayouette why was this merged!? There was a request for changes from @mbologna, and I don't see any commits handling them?

**Pull request merged**